app/views/stories/tagged_articles/_meta.html.erb

```diff
@@ -1,28 +1,28 @@
 <% title_with_timeframe(
-     page_title: @tag.capitalize + (@page.present? && @page > 1 ? t("views.stories.meta.page", page: @page) : ""),
+     page_title: @tag.name.capitalize + (@page.present? && @page > 1 ? t("views.stories.meta.page", page: @page) : ""),
```

Comment:
We seem to use this quite often, do we maybe want to pass it to the view as a local or assign it once like e.g. 

```ruby
<% display_tag_name = @tag.name.capitalize %>
```

and then use this variable?

Reply:
In my experience passing variables to a view rendering is just a little too opaque.  Partially (pun intended?!?) because the `render` method in a view uses the view lookup path, which means it can be a bit harder to track down the upstream caller of the partial and see how we assigned that variable.

I do like the idea of assigning it as a local at the head of the file.  I'll make that tweak.